grobid-core/src/main/java/org/grobid/core/engines/DateParser.java

```diff
@@ -209,65 +218,76 @@ else if ((date.getDayString().charAt(date.getDayString().length() - 1) == '-')
 
     public static final Pattern[] months = {jan, feb, mar, apr, may, jun, jul, aug, sep, oct, nov, dec};
 
-    public void normalize(Date date) {
+    public Date normalize(Date date) {
+        return normalize(date, false);
+    }
+    
+    
+    public Date normalize(Date date, boolean validate) {
```

Comment:
minor: I'd suggest to have a separately named method instead,e.g. `normalizeDateAndValidate`. Otherwise one would need to be familiar with the function in order to understand what the true in `normalize(..., true)` means.

Reply:
good point